### PR TITLE
Add test, remove repl from common commands help output

### LINKED ISSUE #1287: Add dotnet test to the common command help printout.

### PR DESCRIPTION
**Pull request by blackdwarf**
Add test command to the common commands output. Remove repl, since the latest bits don't have it anymore. 

Fix #1287 

/cc @livarcocc @piotrpMSFT

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/1530)

### CODE REVIEW

**Comment by blackdwarf** [comment]
@dotnet-bot test CentOS7.1 Debug Build please

**Comment by blackdwarf** [comment]
@dotnet-bot test CentOS7.1 Release Build please

**Comment by davidfowl** [comment]
Nobody removed repl?

**Comment by tannergooding** [comment]
FYI. @ManishJayaswal

**Comment by ManishJayaswal** [comment]
@blackdwarf @piotrpMSFT This just removes it from help. Has the repl been removed from the default installer?

@richlander  - Additionally we still need to figure out how users can get tools that are not included in the default install.

**Comment by blackdwarf** [comment]
OK, so nobody removed repl, @davidfowl is right, its still there in the source. However, why am I then getting "No executable found matching command "dotnet-repl-csi" when I run `dotnet repl`?

**Comment by piotrpMSFT** [comment]
@blackdwarf can you assign me the issue to remove repl?

File: src/dotnet/commands/dotnet-help/HelpCommand.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public class HelpCommand
   build         Builds a .NET project
   publish       Publishes a .NET project for deployment (including the runtime)
```
Comment (piotrpMSFT) [inline src/dotnet/commands/dotnet-help/HelpCommand.cs]: The order of these seems wrong. Can we alphebatize or put these in order of workflow? Publish is the outlier in that sort order.

**Comment by ManishJayaswal** [comment]
@blackdwarf  @piotrpMSFT My understanding is that we would remove repl from the CLI installer and not from the sources?

**Comment by davidfowl** [comment]
@ManishJayaswal why would we do that?

**Comment by ManishJayaswal** [comment]
@davidfowl I believe we wanted to do that to reduce the size of the default CLI installer. There is an open issue to figure out how users are going to acquire tools which are not included in the default installer.

FYI @richlander

**Comment by piotrpMSFT** [comment]
@blackdwarf conversations regarding REPL aside, it looks like this particular PR can be merged. :shipit:

**Comment by piotrpMSFT** [comment]
@blackdwarf is this still needed?

**Comment by blackdwarf** [comment]
@piotrpMSFT I can still see the repl on 1.0.0-beta-002096 in the help list. Test is, however, added by @livarcocc in another PR i believe.

**Comment by blackdwarf** [comment]
@piotrpMSFT is this OK to be checked in?

**Comment by blackdwarf** [comment]
@piotrpMSFT this ok to be checked in? Do I need to go to shiproom for this?

**Comment by piotrpMSFT** [comment]
:shipit:

**Comment by hickford** [comment]
>dotnet
    .NET Command Line Tools (1.0.0-beta-001598)
    Usage: dotnet [common-options] [command] [arguments]

    Arguments:
      [command]     The command to execute
      [arguments]   Arguments to pass to the command

    Common Options (passed before the command):
      -v|--verbose  Enable verbose output
      --version     Display .NET CLI Version Info

    Common Commands:
      new           Initialize a basic .NET project
      restore       Restore dependencies specified in the .NET project
      build         Builds a .NET project
      publish       Publishes a .NET project for deployment (including the runtime)
      run           Compiles and immediately executes a .NET project
      repl          Launch an interactive session (read, eval, print, loop)
      pack          Creates a NuGet package

That sounds cool, let me try repl:

    >dotnet repl
    No executable found matching command "dotnet-repl-csi"

**Pull request merged**